specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/CommonDefinitions.json

```diff
@@ -908,7 +908,7 @@
       ],
       "properties": {
         "properties": {
-          "$ref": "#/definitions/DaprComponent",
+          "$ref": "#/definitions/DaprComponentProperties",
```

Comment:
Good catch. How did you discover this?

Reply:
I was just trying to pass all the checks and found an error about dapr in this check https://github.com/Azure/azure-rest-api-specs/runs/31530234304